fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -188,19 +187,8 @@ public boolean canCreate(String resourceType, Object resource) {
                       try {
                         fiatService.canCreate(username, resourceType, resource);
                         return true;
-                      } catch (RetrofitError re) {
-                        boolean shouldRetry = true;
-                        if (re.getKind() == RetrofitError.Kind.HTTP) {
-                          switch (HttpStatus.valueOf(re.getResponse().getStatus())) {
-                            case NOT_FOUND:
-                              return false;
-                            case BAD_REQUEST:
-                              shouldRetry = false;
-                          }
-                        }
-                        IntegrationException ie = new IntegrationException(re);
-                        ie.setRetryable(shouldRetry);
-                        throw ie;
+                      } catch (NotFoundException e) {
```

Comment:
OK, this bit is finally sinking in.  Sorry if I pointed you in the wrong direction before.

[SpinnakerRetrofitErrorHandler](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerRetrofitErrorHandler.java) doesn't throw NotFoundException, so I don't think it helps to catch it here.  To get the same behavior as before I think we need to catch SpinnakerHttpException and check for HttpStatus.NOT_FOUND to decide whether to return false.